zap/src/main/java/org/zaproxy/zap/extension/encoder2/EncodeDecodeDialog.java

```diff
@@ -586,6 +612,23 @@ private void updateEncodeDecodeFields() {
             base64DecodeField.setEnabled(false);
         }
 
+        // Base 64 URL
+        try {
+            base64urlEncodeField.setText(
+                    getEncoder().getBase64urlEncode(getInputField().getText()));
+        } catch (NullPointerException | IOException e) {
+            log.error(e.getMessage(), e);
+        }
+
+        try {
+            base64urlDecodeField.setText(
+                    getEncoder().getBase64urlDecode(getInputField().getText()));
+            base64urlDecodeField.setEnabled(base64urlDecodeField.getText().length() > 0);
+        } catch (IOException | IllegalArgumentException e) {
+            base64urlDecodeField.setText(e.getMessage());
+            base64urlDecodeField.setEnabled(false);
```

Comment:
can we log the exception here ?

Reply:
As above, the style is the same as the existing code.

Reply:
Same here.